packages/create-sitecore-jss/src/templates/angular-sxp/.env

```diff
@@ -0,0 +1,2 @@
+# The way in which layout and dictionary data is fetched from Sitecore
+FETCH_WITH=<%- fetchWith %>
```

Comment:
It's not right, since we also have a dictionary/layout services that depend on FETCH_WITH, and still it's related to the base template, empty app should provide both methods, it's not an SXP specific functionality
See: https://github.com/Sitecore/jss/blob/dev/packages/create-sitecore-jss/src/templates/angular/src/app/lib/dictionary-service-factory.ts

Reply:
That condition will only be added if `!locals.xmcloud` (i.e. with SXP setup). 
Empty angular app will never be created, using `angular` template only will create an SXP app, with fetchWith populated

Reply:
let's clarify that today, but in any case we have other code parts that depend on "locals.xmcloud"

Reply:
> but in any case we have other code parts that depend on "locals.xmcloud"
and that's okay. we only need to avoid using prompt values before they were asked and provided.
> I thought we still want to have an empty app (not right now, but moving further)
I don't think so, it would be non-functional with no purpose. Our two main flows are SXP and XMC, this is how we divide functionality, prompts and everything else.

Reply:
As we discussed, the flow that is described is expected, so we keep moving some parts to angular-sxp